unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs

```diff
@@ -429,7 +429,7 @@ public void SetEntityParent(string entityId, string parentId)
                 }
             }
 
-            if (parentId == "0")
+            if (parentId == "0".GetHashCode())
```

Comment:
We should do special code for parent entity, the ID of the root entity will be 0 in the new ECS as it is in the current one. `"0".GetHashCode() != 0`

Reply:
I don't know if I understood correctly what you mean there. We need to maintain retrocompatibility with the old ECS, and the old ECS will send the "0" string. Do you mean change the way the parent works?